modules/fedimint-wallet/src/lib.rs

```diff
@@ -521,18 +522,17 @@ impl FederationModule for Wallet {
             },
             api_endpoint! {
                 "/peg_out_fees",
-                async |module: &Wallet, params: (Address, u64)| -> Option<PegOutFees> {
+                async |module: &Wallet, params: (Address, u64)| -> PegOutFees {
                     let (address, sats) = params;
-                    let consensus = module.current_round_consensus().unwrap();
+                    let consensus = module.current_round_consensus().ok_or_else(||ApiError::new(400, "could not find fee-consensus".to_string()))?;
```

Comment:
we could also return `Result` instead of `Option` and have a `EntityNotInDB` `WalletError` ?